report_thesis/src/sections/background.tex

```diff
@@ -100,4 +93,37 @@ \subsection{Baseline \& Replica}
 While the team decided to retain \gls{pls} for analyzing certain oxides, \gls{ica} was entirely discontinued.
 Instead, models based on \gls{gbr}, \gls{rf}, and \gls{lasso} were selected for other oxides.
 This decision reinforces our finding that \gls{ica} regression models fall short in accurately predicting the composition of major oxides in geological samples.
-Consistent with our observations, \gls{gbr} was also identified as a high-performing model in their analyses.
\ No newline at end of file
+Consistent with our observations, \gls{gbr} was also identified as a high-performing model in their analyses.
+
+\subsection{Data Normalization}\label{sec:data_normalization}
+The \gls{chemcam} instrument consists of three spectrometers, each producing 2048 channels.
+For data normalization, we follow the approach taken by the SuperCam team and normalize across individual spectrometers' wavelength ranges, a process known as \textit{Norm 3}~\cite{andersonPostlandingMajorElement2022}.
+This method ensures that the wavelength intensities measured by each spectrometer is normalized independently.
+
+Figure~\ref{fig:spectral_plot} shows a spectral plot of the \gls{ccs} data for the \textit{ultramafic} sample, illustrating the three distinct spectral regions, each captured by one of the three spectrometers. Specifically, one spectrometer captures the \gls{uv} region, another captures the \gls{vio} region, and the third captures the \gls{vnir} region.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.5\textwidth]{images/spectral_plot.png}
+	\caption{Spectral plot of the \gls{ccs} data for the \textit{ultramafic} sample. The wavelengths represent the spectral channels.}
```

Comment:
```suggestion
	\caption{Spectral plot of the \gls{ccs} data for the \textit{ultramafic} sample. The wavelengths represent the spectral regions.}
```

Reply:
1wvl = 1 channel, so not regions but channels